src/cable/websocket_pinger.cr

```diff
@@ -23,10 +24,15 @@ module Cable
     end
 
     def initialize(@socket : HTTP::WebSocket)
-      Tasker.every(Cable::WebsocketPinger.seconds.seconds) do
+      @task = Tasker.every(Cable::WebsocketPinger.seconds.seconds) do
+        Logger.debug { "Pinging websocket" }
```

Comment:
I can remove this line if desired

Reply:
makes sense, probably would generate a log of log that brings not much value, 500 clients connected, 500 lines every second